eng/native/configureplatform.cmake

```diff
@@ -11,9 +11,9 @@ set(PRERELEASE 1)
 #     - for windows we use the passed in parameter to CMAKE to determine build arch
 #----------------------------------------
 set(CLR_CMAKE_HOST_OS ${CMAKE_SYSTEM_NAME})
-if(CLR_CMAKE_HOST_OS STREQUAL Linux OR CLR_CMAKE_HOST_OS STREQUAL Android)
+if(CLR_CMAKE_HOST_OS STREQUAL Linux)
```

Comment:
Are the changes in configureplatform.cmake really required? We end up setting exactly the same values as before.

Reply:
Not strictly required but we skipped basically 90% of the logic in here so I felt it's clearer to separate Android out.